src/Compilers/CSharp/Portable/Generated/Syntax.xml.Internal.Generated.cs

```diff
@@ -6153,31 +5831,25 @@ internal AnonymousMethodExpressionSyntax(ObjectReader reader)
         : base(reader)
     {
       this.SlotCount = 5;
-      var asyncKeyword = (SyntaxToken)reader.ReadValue();
+      var asyncKeyword = (SyntaxToken?)reader.ReadValue();
       if (asyncKeyword != null)
```

Comment:
should we remove these checks as well?  to keep things simple and consistent between constructor and reader?